CHANGELOG.md

```diff
@@ -18,7 +18,7 @@
 - Improve startup and shutdown messages ([PR 2770](https://github.com/input-output-hk/daedalus/pull/2770))
 - Updated `cardano-wallet` to version `2021-11-11` ([PR 2765](https://github.com/input-output-hk/daedalus/pull/2765))
 - Added jest library for unit testing ([PR 2633](https://github.com/input-output-hk/daedalus/pull/2633))
-- Updated `cardano-launcher` to version `0.20211105.1` and added Cardano Node RTS flags which improve resource usage ([PR 2735](https://github.com/input-output-hk/daedalus/pull/2735), [PR 2741](https://github.com/input-output-hk/daedalus/pull/2741))
+- Updated `cardano-launcher` to version `0.20211105.1`
```

Comment:
@danielmain should this PR have a changelog entry?
https://github.com/input-output-hk/daedalus/pull/2741